custom-instrumentation/attributes-and-events/README.md

```diff
@@ -0,0 +1,23 @@
+# Sample Attributes & Events Application
+
+This example provides an application to demonstrate how to share custom attributes and events. This application is helpful for users who want to send custom attributes and events in a web framework transaction.
+
+## Getting Started
+
+1. Clone or fork this repository.
+2. Navigate to this example's sub directory
+   ```
+   cd newrelic-node-examples/custom-instrumentation/attributes-and-events
+   ```
+3. Install dependencies and run application.
+   ```
+   npm install
+   cp env.sample .env
+   # Fill out `NEW_RELIC_LICENSE_KEY` in .env and save 
+   # Start the application
+   npm start
+   ```
+4. Make requests to the application. See index.js for full list of accepted route names.
+   ```
+   curl -X POST http://localhost:3000/custom-attribute
```

Comment:
i would include all routes just for convenience sake

Reply:
I'd also include how to view the data getting created. The first 4 routes just attach an attribute to the route handler span.  The custom event you can query via `FROM `my_app:my_event` SELECT *`